presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/MergeAdjacentWindows.java

```diff
@@ -19,9 +19,12 @@
 import com.facebook.presto.sql.planner.plan.PlanNode;
 import com.facebook.presto.sql.planner.plan.WindowNode;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
 
 import java.util.Optional;
 
+import static com.facebook.presto.sql.planner.iterative.rule.Util.pullUnaryNodeAboveProjects;
+import static com.facebook.presto.sql.planner.iterative.rule.Util.restrictOutputs;
 import static com.facebook.presto.sql.planner.optimizations.WindowNodeUtil.dependsOn;
 
 public class MergeAdjacentWindows
```

Comment:
Maybe this rule and the swap rule should belong to the same `RuleSet`? What do you think?